### PR TITLE
Current year leaderboard

### PR DESCRIPTION
**Pull request by cauenapier**

### CODE REVIEW

File: aim4/challenges/views.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 from random import shuffle
+import datetime
```
Comment (joaofogoncalves) [inline aim4/challenges/views.py:2]: Unecessary import

**Review by joaofogoncalves — CHANGES_REQUESTED** [review]
I Don't think we need an extra tab, just another column on same list, and the possibility to order it.
https://mdbootstrap.com/docs/b4/jquery/tables/sort/

**Comment by cauenapier** [comment]
Just out of curiosity, @joaofogoncalves 
I was divided between writing the total distances as a property or making it a database attribute. How would you decide between the two options?

My rationale was: if pages loads are more often then new entries (new activities), then it is better to add this as a database attribute, since we will just calculate the new total distance when a new activity is added.
On the other hand, if page loads (which is the case now) is less often then new entries, it might be better to leave the calculation of the total distance whenever we open up the page.

Is this line of thought correct? Is there something else that should be taken into consideration?

**Comment by joaofogoncalves** [comment]
That is the correct thinking, but in such low loads as we have right now, not very important.

**Pull request merged**